bootloader/Kconfig

```diff
@@ -0,0 +1,27 @@
+mainmenu "SC-OBC Module A1 Bootloader"
+
+menu "Application"
+
+config SCOBC_A1_GOLDEN_CFG_MEM_ADDR
+	hex "The address of the config memory where the bootloader loads the Golden Flight software"
```

Comment:
"Golden Flight software", why "G" and "F" are captalized but not "s"? Any reason?

Reply:
No reason.. 
It's better to `Golden Flight Software`.